awslambda/deletesnippet/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+console.log('Loading function');
+
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({ apiVersion: '2006-03-01' });
+
+
+exports.handler = (event, context, callback) => {
+    console.log(JSON.stringify(event));
+    var user_id = event.pathParameters.user_id;
+    var snippetKey = event.pathParameters.snippet_id;
```

Comment:
Use `const` in place of `var`. Also, 12 and 13 can be written as
`const {user_id, snippet_id as snippetKey} = event.pathParameters`